smrf/tests/__init__.py

```diff
@@ -1 +1,3 @@
 # -*- coding: utf-8 -*-
+
+non_distributed_variables = ['time', 'x', 'y', 'projection']
```

Comment:
I would stick with how the variables are capitalized in the netCDF

Reply:
Reading this a second time, I think I misunderstood your comment.

Yes - to capitalization and also could have a better home in the
`SMRFTestCase` base class since we are defining a similar constant on line#25 there

Reply:
Good advice all around. I'll move it and capitalize it.